src/Archives/ClmFile.cpp

```diff
@@ -272,16 +272,17 @@ namespace Archives
 	{
 		uint64_t offset = headerSize + names.size() * sizeof(IndexEntry);
 		for (std::size_t i = 0; i < names.size(); ++i)
-		{
-			// Disable MSVC warning 'strncpy': This function or variable may be unsafe. Consider using strncpy_s instead.
+		{      
+      // Disable MSVC warning 'strncpy': This function or variable may be unsafe. Consider using strncpy_s instead.
```

Comment:
This merge commit seems to have introduced some whitespace errors. There are spaces here instead of the usual tabs used throughout the rest of the file.

Other blocks of code follow with the same problem.